packages/webapp/src/containers/Filter/types.ts

```diff
@@ -14,7 +14,14 @@
  */
 
 // In tempStateReducer > filterReducer
-export type ReduxFilterEntity<FilterKey extends string = string> = Record<FilterKey, FilterState>;
+export type ReduxFilterEntity<FilterKey extends string = string> = Record<FilterKey, FilterState> &
```

Comment:
Unlike the other filters, the date filters have a string type. In order to set the filter inactive the filter value needs to be set to undefined rather than using an `active` flag. I need to explicitly specify this type here in order to reset to undefined in `FilterPage/index.tsx`.